description_templates/golden_test.go

```diff
@@ -2,21 +2,32 @@ package descriptiontemplates_test
 
 import (
 	"context"
+	"flag"
+	"io"
 	"io/fs"
 	"os"
 	"path/filepath"
 	"strings"
 	"testing"
 
 	"github.com/appuio/appuio-cloud-reporting/pkg/invoice"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"github.com/vshn/appuio-odoo-adapter/invoice/desctmpl"
 )
 
 const extension = ".gotmpl"
 
+var (
+	updateGolden = flag.Bool("update", false, "update the golden files of this test")
+)
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+	os.Exit(m.Run())
+}
+
 func TestGenerateGolden(t *testing.T) {
-	os.RemoveAll("golden")
```

Comment:
This was added to have a cleanup function if a template gets moved/deleted. Maybe execute it if `-update` is given? The test should still pass.